packages/experiments/src/components/SelectedItemsList/SelectedPeopleList/Items/SelectedItemWithContextMenu.tsx

```diff
@@ -0,0 +1,63 @@
+/* tslint:disable */
+import * as React from 'react';
+/* tslint:enable */
+import { BaseComponent, autobind } from '../../../../Utilities';
+import { IExtendedPersonaProps } from '../SelectedPeopleList';
+import { ContextualMenu, DirectionalHint } from 'office-ui-fabric-react/lib/ContextualMenu';
+import { IContextualMenuItem } from 'office-ui-fabric-react/lib/ContextualMenu';
+import { IBaseProps } from 'office-ui-fabric-react/lib/Utilities';
+
+export interface IPeoplePickerItemState {
+  contextualMenuVisible: boolean;
+}
+
+export interface ISelectedItemWithContextMenuProps extends IBaseProps {
+  renderedItem: JSX.Element;
+  beginEditing?: (item: IExtendedPersonaProps) => void;
+  menuItems: IContextualMenuItem[];
+  item: IExtendedPersonaProps;
+}
+
+export class SelectedItemWithContextMenu extends BaseComponent<ISelectedItemWithContextMenuProps, IPeoplePickerItemState> {
```

Comment:
This almost feels too specific, could you style a button with contextualmenuitems to meet your needs?

Reply:
You're right. I will revisit this and try and create a better abstraction to fit my needs